mindsdb/migrations/versions/2022-11-07_1e60096fc817_predictor_version.py

```diff
@@ -0,0 +1,56 @@
+"""predictor_version
+
+Revision ID: 1e60096fc817
+Revises: 43c52d23845a
+Create Date: 2022-11-07 16:43:47.301692
+
+"""
+from alembic import op
+import sqlalchemy as sa
+import mindsdb.interfaces.storage.db as db
+
+
+
+# revision identifiers, used by Alembic.
+revision = '1e60096fc817'
+down_revision = '43c52d23845a'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table('predictor', schema=None) as batch_op:
+        batch_op.add_column(sa.Column('label', sa.String(), nullable=True))
+        batch_op.add_column(sa.Column('version', sa.Integer(), nullable=True))
+
+    # update current predictor versions
+    conn = op.get_bind()
+    session = sa.orm.Session(bind=conn)
+
+    key0 = (None, None, None)
+    for p in session.query(db.Predictor)\
+            .order_by(db.Predictor.name, db.Predictor.id)\
```

Comment:
deleted_at == null() ?

Reply:
we assign version to deleted predictors too